src/tests/system/tests/test_sss_cache.py

```diff
@@ -43,3 +46,67 @@ def test_sss_cache__cache_expire_message(client: Client):
     for cmd in ("sss_cache -U", "sss_cache -G", "sss_cache -E", "sss_cache --user=nonexisting"):
         res = client.host.ssh.run(cmd)
         assert "No domains configured, fatal error!" not in res.stdout
+
+
+@pytest.mark.importance("critical")
+@pytest.mark.cache
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_sss_cache__background_refresh(client: Client, provider: GenericProvider):
+    """
+    :title: Ensuring ldb cache data is refreshed correctly
+    :setup:
+        1. Create provider user
+        2. Create provider group
+        3. Create provider netgroup
+        4. Configure SSSD and set 'entry_cache_timeout' to 4 and 'refresh_expired_interval' to 8
+        5. Restart SSSD
+        6. Populate the cache by performing 'getent' on the user, group and netgroup
+    :steps:
+        1. Search for user, group and netgroup lastUpdate and dataExpireTimestamp in the ldb database
+        2. Wait 10 seconds and search for all timestamp in the cache again
+    :expectedresults:
+        1. The 'dataExpireTimestamp' value equals the 'lastUpdate + entry_cache_timeout' value
+        2. User, group and netgroup 'lastUpdate' timestamp value has been refreshed
+    """
+    user = provider.user("test_user").add()
+    provider.group("test_group").add().add_member(user)
+    provider.netgroup("test_netgroup").add().add_member(user=user)
+
+    domain = client.sssd.default_domain
+    entry_cache_timeout = 4
+    refresh_expired_interval = 8
```

Comment:
I know we already discussed this point, but would it be possible to set these two values to lower numbers? Maybe 1 and 2? This way we can reduce the sleep time even more.

Reply:
It seems to be stable with 1, 2 and a sleep of 5.

Reply:
Perfect